parm/config/config.resources

```diff
@@ -335,6 +385,7 @@ elif [ $step = "vrfy" ]; then
     export wtime_vrfy="03:00:00"
     export wtime_vrfy_gfs="06:00:00"
     export npe_vrfy=3
+    [[ "$machine" == "S4" ]] && export npe_vrfy=1
```

Comment:
We really need to know what is the drive behind some of these special S4 resource requests.